account/service/tenant.go

```diff
@@ -16,28 +16,28 @@ import (
 	"github.com/satori/go.uuid"
 )
 
-type tenantConfig interface {
-	GetTenantServiceURL() string
-}
-
-// Tenant represents Tenant Service
-type Tenant interface {
+// TenantService represents the Tenant service
+type TenantService interface {
```

Comment:
Good cleanup of old inconsistent naming.

Reply:
thanks ;)